### PR TITLE
taxonomy: categories - Tartinades d'oeufs

### PR DESCRIPTION
**Pull request by moon-rabbitOFF**

### CODE REVIEW

File: taxonomies/categories.txt [hunk 1]
```diff
@@ -85144,6 +85144,12 @@ pt:Produtos salgados para barrar
 food_groups:en: en:Salty and fatty products
 pnns_group_2:en:Salty and fatty products
 
+# This product is really popular in Belgium were it is called "Salade d'oeufs". Please don't put in the category "Prepared salads", because it is a spread not a meal
+<en:Salted spreads
+en:Egg spreads
+de:Eiaufstriche
+fr:Tartinades d'œufs, d'oeufs
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:85151]:
```suggestion
fr:Tartinades d'œufs, tartinades d'oeufs
```

Comment (teolemon) [reply taxonomies/categories.txt:85151]: Tartinade aux oeufs

Comment (moon-rabbitOFF) [reply taxonomies/categories.txt:85151]: I modified it

**Review by stephanegigandet — APPROVED** [review]
Thanks!

**Pull request merged**